packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewer.js

```diff
@@ -47,33 +51,93 @@ function SingleImageViewer({
   zoomControlFn = null,
   zooming = true
 }) {
+  const styledSVGRef = useRef(null)
+  const [svgDimensions, setSvgDimensions] = useState({ width: '100%', height: '100%' })
+  
+  const [showZoomHelper, setShowZoomHelper] = useState(false)
+  const [fadingOut, setFadingOut] = useState(false)
+
   useEffect(function onMount() {
     enableRotation()
   }, [])
 
+  useEffect(function onDimensionChange() {
+    // Update dimensions after component has mounted and whenever natural width/height changes
+    if (styledSVGRef.current) {
+      function updateDimensions() {
+        setSvgDimensions({
+          width: `${styledSVGRef.current.clientWidth}px`,
+          height: `${styledSVGRef.current.clientHeight}px`
+        })
+      }
+      
+      updateDimensions()
+    }
+  }, [naturalWidth, naturalHeight, styledSVGRef])
+
+  // Handle the first scroll event
+  function handleFirstScroll() {
+    if (allowsScrolling && zooming) {
+      setShowZoomHelper(true)
+      
+      // Set fading out after 3 seconds
+      setTimeout(() => {
+        setFadingOut(true)
+      }, 3000)
+      
+      // Hide completely after animation completes (300ms animation)
+      setTimeout(() => {
+        setShowZoomHelper(false)
+        setFadingOut(false)
+      }, 3300)
+    }
+  }
 
   const maxHeight = limitSubjectHeight ? `min(${naturalHeight}px, 90vh)` : null
   const maxWidth = limitSubjectHeight ? `${naturalWidth}px` : '100%'
 
+  // If the image is landscape, set the top margin to 60px
+  // If the image is portrait, set the top margin to 240px (ImageToolbar with fewest options height of 360px - ZoomHelperOverlay message box height of 120px)
+  const zoomHelperMessageTop = naturalHeight > naturalWidth ? '240px' : '60px'
+
   return (
     <>
       {zoomControlFn && (
         <ZoomControlButton onClick={zoomControlFn} zooming={zooming} />
       )}
-      <Box align="flex-end" animation="fadeIn" overflow="hidden" width="100%">
+      <Relative
+        align='flex-end'
+        animation='fadeIn'
+        overflow='hidden'
+        width='100%'
+      >
+        {showZoomHelper && (
+          <ZoomHelperOverlay
+            fadingOut={fadingOut}
+            height={svgDimensions.height}
+            width={svgDimensions.width}
+            zoomHelperMessageTop={zoomHelperMessageTop}
+          />
+        )}
         <StyledSVG
+          ref={styledSVGRef}
           aria-labelledby={title?.id}
+          aria-describedby={allowsScrolling ? 'scrolling-info' : undefined}
```

Comment:
I'm not sure about this. I tried using aria-live region, Grommet's `AnnounceContext`, and other approaches, but none worked, and I'm not sure what I was doing wrong. Regardless, I think this is a reasonable approach, and I've confirmed it's as I expect with VoiceOver, but very much open to suggestion.

Reply:
I suppose eventually #6809 will compliment this.

Reply:
Screen reader users use the keyboard to navigate the page, not the mouse, so I’m not sure how this announcement would be triggered by someone using a screen reader.

Reply:
`aria-describedby` isn’t widely supported either, AFAIK.

>Definitely do not rely on the accessible description to convey critical information (such as [instructions](https://adrianroselli.com/2019/10/stop-giving-control-hints-to-screen-readers.html)).

https://adrianroselli.com/2022/04/accessible-description-exposure.html

Reply:
If you want a quick patch for JAWS and NVDA, `aria-labelledby` accepts a list of IDREFs, so this would concatenate the title and description into the accName:

```jsx
<svg aria-labelledby={`${title.id} ${allowsScrolling ? 'scrolling-info' :''}` >
```

But yeah, longer term maybe use some personas for disabled volunteers to generate some quick user stories for the zoom feature with assistive technology here. I'm not 100% convinced that the message about mouse wheel zoom is useful to screen reader users, but I could be wrong.